### DISCUSSION TITLE
blog/meshtastic-emoji-short-names/

### DISCUSSION THREAD

**Discussion by giscus** [meshtastic/Discussions#112] [Blog Post Comments]
# blog/meshtastic-emoji-short-names/

Discover how emojis can personalize your Meshtastic experience, from Short Names and waypoints to expressive OLED messages.

https://meshtastic.org/blog/meshtastic-emoji-short-names/

**Comment by AA7US** [comment]
What fun! 🤩

**Comment by lykamspam** [comment]
Is trouble with MQTT  msh/[emoji] for example add solar panel, wall power etc. sorry this is too short for many emoji. I cant add for example my country 2 chars and emoji

**Comment by Coopersmith-24601** [comment]
It should be noted there is a long time standing issue with the "Heart" emoji displaying correctly on OLED screens when sent from a node using the iOS app.  I assume it's the same coding issue which was fixed with the "Heart Suit" emoji, but somehow still hasn't been corrected in Tap Back.
